lms/djangoapps/program_enrollments/api/v1/views.py

```diff
@@ -643,6 +664,8 @@ def get(self, request, program_uuid=None, course_id=None):
         serializer = ProgramCourseEnrollmentListSerializer(paginated_enrollments, many=True)
         return self.get_paginated_response(serializer.data)
 
+    @verify_program_exists
```

Comment:
You might want to switch the order of these, so that we check for the program existence first, since `verify_course_exists_and_in_program` assumes that the program exists.

Reply:
I'm pretty sure this is the correct order, the tests fail when they're the other way around for that exact reason

Reply:
Ah, yup, you're right - you're doing your validation before you return the result of the wrapped function, my mistake.